CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Welcome to the Momento Go-Redis contributing guide :wave:
+
+Thank you for taking your time to contribute to our Momento `go-redis` wrapper!
+<br/>
+This guide will provide you information to start your own development and testing.
+<br/>
+Happy coding :dancer:
+<br/>
+
+## Requirements :coffee:
+
+- Go 1.18 or above is required https://go.dev/doc/install
+- A Momento auth token is required, you can generate one using the [Momento Console](https://console.gomomento.com)
+
+<br/>
+
+## Build :computer:
+
+```
+make build
+```
+
+<br/>
+
+## Formatting and Tidy :flashlight:
+
+```
+make lint
+```
+
+<br/>
+
+## Tests :zap:
+
+### Run integration tests against Momento 
+
+```bash
+TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
+make test-momento
```

Comment:
```suggestion
TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN> make test-momento
```

or

```
export TEST_AUTH_TOKEN=<YOUR_AUTH_TOKEN>
make test-momento
```

Reply:
it actually works on a single line as well! do you mean more for readability?

Reply:
If I just issue those commands as you have them there, one after another without using `export`, the tests fail with an `invalid memory address or nil pointer dereference` when it tries to use the credential provider.

Reply:
oh interesting! changed